deploy-as-code/helm/environments/works-dev.yaml

```diff
@@ -177,6 +177,7 @@ estimate-service:
   java-enable-debug: true
   estimate-default-limit: 100
   estimate-search-max-limit: 200
+  estimate-workflow-module-name: "estimate-approval-2"

```

Comment:
This isn't module name. In workflow, module name attribute is used for a different purpose.

Reply:
corrected.